khmer/_khmermodule.cc

```diff
@@ -4480,6 +4578,38 @@ static PyMethodDef khmer_hllcounter_methods[] = {
     {NULL} /* Sentinel */
 };
 
+static PyGetSetDef khmer_hllcounter_getseters[] = {
+    {
+        (char const *)"alpha",
+        (getter)hllcounter_getalpha, NULL,
+        (char const *)"alpha constant for this HLL counter.",
+        NULL
+    },
+    {
+        (char const *)"error_rate",
+        (getter)hllcounter_get_erate, (setter)hllcounter_set_erate,
+        (char const *)"Error rate for this HLL counter. "
+        "Can be changed prior to first counting, but becomes read-only after "
+        "that (raising AttributeError)",
+        NULL
+    },
+    {
+        (char const *)"ksize",
+        (getter)hllcounter_get_ksize, (setter)hllcounter_set_ksize,
+        (char const *)"k-mer size for this HLL counter."
+        "Can be changed prior to first counting, but becomes read-only after "
+        "that (raising AttributeError)",
+        NULL
+    },
+    {
+        (char const *)"counters",
+        (getter)hllcounter_getcounters, NULL,
+        (char const *)"Read-only internal counters.",
+        NULL
+    },
+    {NULL} /* Sentinel */
+};
```

Comment:
I get multiple warnings for the same line number so I'm guessing it is all
the conversions within the code block.

On Tue, Feb 24, 2015, 19:06 Luiz Irber <notifications@github.com> wrote:

> In khmer/_khmermodule.cc
> <https://github.com/ged-lab/khmer/pull/738#discussion_r25306902>:
>
> > +    {
> > +        (char const *)"ksize",
> > +        (getter)hllcounter_get_ksize, (setter)hllcounter_set_ksize,
> > +        (char const *)"k-mer size for this HLL counter."
> > +        "Can be changed prior to first counting, but becomes read-only after "
> > +        "that (raising AttributeError)",
> > +        NULL
> > +    },
> > +    {
> > +        (char const *)"counters",
> > +        (getter)hllcounter_getcounters, NULL,
> > +        (char const *)"Read-only internal counters.",
> > +        NULL
> > +    },
> > +    {NULL} /* Sentinel */
> > +};
>
> So, I don't really get *where* is it coming from. Is it from the sentinel?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ged-lab/khmer/pull/738/files#r25306902>.
>